Python2's ConfigParser module used by setuptools throws parsing errors when
encountering tab characters.

--- Kconfiglib-12.12.1/setup.cfg
+++ Kconfiglib-12.12.1/setup.cfg
@@ -1,6 +1,6 @@
 [bdist_wheel]
-	# We support both Python 2 and Python 3
-	universal = 1
+# We support both Python 2 and Python 3
+universal = 1
 [metadata]
-	# Include the license file in wheels
-	license_file = LICENSE.txt
+# Include the license file in wheels
+license_file = LICENSE.txt
